### PR TITLE
✨ Freestar RTC Module

### PR DESCRIPTION
**Pull request by nicgallardo**
✨ Freestar RTC Module
- Implement of Freestar RTC Module in `extensions/amp-a4a/0.1/callout-vendors.js`
- Addition of Freestar as a vendor in `extensions/amp-a4a/rtc-publisher-implementation-guide.md` and `extensions/amp-a4a/rtc-documentation.md`
- Example implementation `examples/freestar.amp.html`

### CODE REVIEW

**Comment by premesh-freestar** [comment]
@googlebot I signed it!

**Comment by calebcordry** [comment]
Thanks for contributing! It looks like the commits are from a gmail address, not sure if thats what you signed the CLA with? Let me know if you have problems figuring it out.

File: examples/ads.amp.esm.html [hunk 1]
```diff
@@ -2163,7 +2163,6 @@ <h2>Teads</h2>
       data-pid="42266"
       layout="responsive">
   </amp-ad>
-
```
Comment (calebcordry) [inline examples/ads.amp.esm.html:2166]: please remove this change.

File: examples/freestar.amp.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<!doctype html>
```
Comment (calebcordry) [inline examples/freestar.amp.html:1]: can you move the ad tag to `ads.amp.html`, let me know if there is reason you think you need your own example file.

File: extensions/amp-a4a/rtc-publisher-implementation-guide.md [hunk 3]
```diff
@@ -36,6 +36,7 @@ To use RTC, you must meet the following requirements:
 - Yieldbot
 - Kargo
 - Yieldlab
+- [Freestar](https://freestar.com)
```
Comment (calebcordry) [inline extensions/amp-a4a/rtc-publisher-implementation-guide.md:39]: nit: alphabetical order

File: extensions/amp-a4a/rtc-documentation.md [hunk 4]
```diff
@@ -141,6 +141,7 @@ The `errorReportingUrl` property is optional. The only available macros are ERRO
 - Yieldbot
 - Kargo
 - Yieldlab
+- [Freestar](https://freestar.com)
```
Comment (calebcordry) [inline extensions/amp-a4a/rtc-documentation.md:144]: nit: alphabetical order

Comment (calebcordry) [reply extensions/amp-a4a/rtc-documentation.md:144]: you can also move Kargo here and below.

**Comment by nicgallardo** [comment]
@calebcordry I have two email address attached to my Github account. The primary was my gmail but I switched to my corporate account. Should I close the PR and redo my contribution with my corporate account instead to bypass this issue?

**Comment by calebcordry** [comment]
If you are familiar with git rebase you can rewrite the commit author to the correct email. I found a guide here: https://www.git-tower.com/learn/git/faq/change-author-name-email 

If you have problems with that opening a new PR is fine too.

**Pull request closed without merging**